docs/docs-content/cluster-templates/create-cluster-template-policies/create-cluster-template-policies.md

```diff
@@ -0,0 +1,102 @@
+| ------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| **Name**            | Name of the cluster template. Each template name must be unique.                                                                                                                                                                                                                                                                                                               |
+| **Env**             | Infrastructure environment of the cluster template. This is based on the infrastructure or managed Kubernetes provider selected when creating the linked [infrastructure](../../profiles/cluster-profiles/create-cluster-profiles/create-infrastructure-profile.md) or [full cluster profile](../../profiles/cluster-profiles/create-cluster-profiles/create-full-profile.md). |
+| **Last Modified**   | Date the template was last modified.                                                                                                                                                                                                                                                                                                                                           |
+| **In use clusters** | Specifies if the template is linked to an active cluster. If a template is linked, select the drawer icon beside it to view details about the template. For more information, refer to our [Modify Cluster Templates](../modify-cluster-templates.md) guide.                                                                                                                   |
+| **Scope**           | Specifies if the template is part of the tenant or project scope.                                                                                                                                                                                                                                                                                                              |
+| **Tags**            | Tags assigned to the template.                                                                                                                                                                                                                                                                                                                                                 |
```

Comment:
Can we simply say that the same columns are available?

Reply:
Sorry @cloudmaniac, not sure I understand this comment. The table columns are different to the Policies Tab ones.